src/nidmm/metadata/enums.py

```diff
@@ -1,148 +1,254 @@
-        {'name': 'DMM_MODE', 'value': 0},
-        {'name': 'WAVEFORM_MODE', 'value': 1},
-    ],
-
-    'Slope': [
-        {'name': 'POSITIVE', 'value': 0},
-        {'name': 'NEGATIVE', 'value': 1},
-    ],
-
-    'TemperatureRTDType': [
-        {'name': 'CustomRTD', 'value': 0},
-        {'name': 'PT3750', 'value': 1},
-        {'name': 'PT3851', 'value': 2},
-        {'name': 'PT3911', 'value': 3},
-        {'name': 'PT3916', 'value': 4},
-        {'name': 'PT3920', 'value': 5},
-        {'name': 'PT3928', 'value': 6},
-    ],
-
-    'TemperatureThermistorType': [
-        {'name': 'THERMISTOR_CUSTOM', 'value': 0},
-        {'name': 'THERMISTOR_44004', 'value': 1},
-        {'name': 'THERMISTOR_44006', 'value': 2},
-        {'name': 'THERMISTOR_44007', 'value': 3},
-    ],
-
-    'TemperatureThermocoupleReferenceJunctionType': [
-        {'name': 'Fixed', 'value': 2},
-    ],
-
-    'TemperatureThermocoupleType': [
-        {'name': 'B', 'value':  1},
-        {'name': 'E', 'value':  4},
-        {'name': 'J', 'value':  6},
-        {'name': 'K', 'value':  7},
-        {'name': 'N', 'value':  8},
-        {'name': 'R', 'value':  9},
-        {'name': 'S', 'value': 10},
-        {'name': 'T', 'value': 11},
-    ],
-
-    'TemperatureTransducerType': [
-        {'name': 'THERMOCOUPLE', 'value': 1},
-        {'name': 'THERMISTOR', 'value': 2},
-        # TODO(marcoskirsch): annoying, C constants start with numbers. Replaced with spelled out values.
-        {'name': 'TWO_WIRE_RTD', 'value': 3},
-        {'name': 'FOUR_WIRE_RTD', 'value': 4},
-    ],
-
-    'WaveformCouplingMode': [
-        {'name': 'WAVEFORM_COUPLING_AC', 'value': 0},
-        {'name': 'WAVEFORM_COUPLING_DC', 'value': 1},
-    ],
-
-    'Terminal': [
-        {'name': 'NONE', 'value': -1},
-        {'name': 'IMMEDIATE', 'value': 1},
-        {'name': 'EXTERNAL', 'value': 2},
-        {'name': 'SOFTWARE_TRIG', 'value': 3},
-        {'name': 'PXI_TRIG0', 'value': 111},
-        {'name': 'PXI_TRIG1', 'value': 112},
-        {'name': 'PXI_TRIG2', 'value': 113},
-        {'name': 'PXI_TRIG3', 'value': 114},
-        {'name': 'PXI_TRIG4', 'value': 115},
-        {'name': 'PXI_TRIG5', 'value': 116},
-        {'name': 'PXI_TRIG6', 'value': 117},
-        {'name': 'PXI_TRIG7', 'value': 118},
-        {'name': 'PXI_STAR', 'value': 131},
-        #{'name': 'LBR_TRIG0', 'value':1003}, # TODO: Internal Trigger Line of a PXI/SCXI Combination Chassis
-        #{'name': 'LBR_TRIG1', 'value':1004}, # TODO: Internal Trigger Line of a PXI/SCXI Combination Chassis
-        {'name': 'AUX_TRIG1', 'value': 1001},
-        {'name': 'INTERVAL', 'value': 10},
-    ],
-
-    'AcquisitionStatus': [
-        {'name': 'RUNNING', 'value': 0},
-        {'name': 'FINISHED_WITH_BACKLOG', 'value': 1},
-        {'name': 'FINISHED_WITH_NO_BACKLOG', 'value': 2},
-        {'name': 'PAUSED', 'value': 3},
-        {'name': 'NO_ACQUISITION_IN_PROGRESS', 'value': 4},
-    ],
+    'ApertureTimeUnits': {
```

Comment:
These were arrays so that we can control the order of the enum values, which we may want to display in an order different than their ascending numeric values.

What was behind this change?